deploy/common/pmem-csi.intel.com_v1beta1_pmemcsideployment_cr.yaml

```diff
@@ -6,7 +6,7 @@ spec:
   deviceMode: "lvm"
   nodeSelector:
     # When using Node Feature Discovery (NFD):
-    feature.node.kubernetes.io/memory-nv.dax: "true"
+    # feature.node.kubernetes.io/memory-nv.dax: "true"
     # When using manual node labeling with that label:
-    # storage: pmem
+    storage: pmem
```

Comment:
Why this change?

Reply:
We shouldn't encourage users to use "storage" as label. It's a bad choice for a label because it isn't namespaced and may conflict with other usage of labels.

We are kind of stuck with it because we promised seamless updates from previous versions, but for someone who is starting out anew (and people using the operator will be in that camp) it makes sense to pick a better label.

The NFD label can also be set manually in case that NFD is not used.

Reply:
> Why this change?

There is a mismatch between the documentation and the example CR, an the example is not usable out of the box in a cluster where no NFD installed. 

> but for someone who is starting out anew (and people using the operator will be in that camp) it makes sense to pick a better label.
> 
> The NFD label can also be set manually in case that NFD is not used.

Then shall we replace the node label name in the OperatorHub documentation as below:

```
**Label the cluster nodes that provide persistent memory device(s)**

$ kubectl label node <your node> feature.node.kubernetes.io/memory-nv.dax="true"

You can skip this step if your cluster is installed with [Node Feature Discovery(NFD)](https://kubernetes-sigs.github.io/node-feature-discovery/stable/get-started/index.html)

```

Reply:
Which documentation is "the documentation"? https://intel.github.io/pmem-csi/latest/docs/install.html#installation-and-setup ?

It introduces the "storage=pmem" label as a "historic" choice. That some other label like the one from NFD is less likely to conflict is not called out explicitly, but could be added.

> Then shall we replace the node label name in the OperatorHub documentation as below:
[...]

Yes, I think that is better.

But how much documentation do we really want to store on OperatorHub? Do we expect that people will be able to use the operator solely based on information there? I'm worried that we'll end up maintaining our documentation in two places.

Reply:
"You can skip this step if your cluster is installed with" -> "You can skip this step if your cluster has NFD installed"

Reply:
> But how much documentation do we really want to store on OperatorHub?

Very minimal. It just has software/hardware requirements and nodes setup as part of pre-requisites.

Reply:
> "You can skip this step if your cluster is installed with" -> "You can skip this step if your cluster has NFD installed"

Done.